File: codewars/JavaScript/7kyu/Check the exam.js
function checkExam(array1, array2) {
    const score = array2.reduce((sum, n, i) => {
      if(n === array1[i]) {
        sum += 4
      } else if (n !== "") {
        sum -= 1
      }
      return sum
    }, 0)
    
    return (score > 0) ? score : 0
  }

File: codewars/Python/7kyu/Check the exam.py
def check_exam(arr1,arr2):
    score = 0
    
    for i,_ in enumerate(arr1):
        if(arr1[i] == arr2[i]):
            score += 4
        elif not (arr2[i] == ""):
            score -= 1
    
    return score if score > 0 else 0